eclean-kernel: add page (#16202)
tldr-pages/tldr: pages/linux/eclean-kernel.md

# eclean-kernel

> Remove old kernels in Gentoo.
> More information: <https://wiki.gentoo.org/wiki/Kernel/Removal#Using_eclean-kernel>.

- List all kernel files:

`sudo eclean-kernel -l`

> - Remove all kernels, but leave the newest two kernels:

Reviewer: ```suggestion
- Remove all kernels except for the two newest ones:
```

`sudo eclean-kernel -n 2`

> - Remove all kernels, but leave the [n]ewest two kernels and [a]sk before removal:

Reviewer: ```suggestion
- Remove all kernels except for the two newest ones and ask before removal:
```

> - Remove all kernels, but leave the newest two kernels and ask before removal:

Reviewer: ```suggestion
- Remove all kernels except for the two newest ones and ask before removal:
```

> `sudo eclean-kernel -a -n 2`

Reviewer 2: ```suggestion
`sudo eclean-kernel {{[-a|--all]}} {{[-n|--num]}} 2`
```
